main.py

```diff
@@ -2,6 +2,36 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres_data = [
+        ("Western",),
+        ("Action",),
+        ("Dramma",)
+    ]
+    for genre_name in genres_data:
+        Genre.objects.create(name=genre_name[0])
+
+    actors_data = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
```

Comment:
There are typos in the actor names 'George Klooney' and 'Kianu Reaves'. These are corrected later in the script, but it's better to fix them in the initial data to avoid unnecessary updates.